content/boards/arduino-mega-2560/index.md

```
@@ -16,6 +16,17 @@
 - 54 digital IO pins, 15 with PWM support.
 - 16 analog inputs (can be used as digital inputs).
 
+| Device | Limit | Notes | 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)